index.html

```diff
@@ -2,13 +2,113 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <title> Taher Haggui </title>
-        <link rel="stylesheet" type="text/css" href="styles/template2.css">
+        <title> Jamstack </title>
+        <link rel="stylesheet" type="text/css" href="styles/home_style.css">
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-        <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;1,100;1,600&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400&display=swap" rel="stylesheet">
+        <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet" type="text/css"/>
         <meta name="viewport" content="width=device-width, initial-scale=1">
     </head>
     <body>
+        <section class="home-page">
+          <header>
+            <nav class="desktop">
```

Comment:
[ OPTIONAL ] User when clicking on `Home` or `About` on the hamburger menu, page shows but then hamburger gets stuck and overall web page is stuck. So this is not good usability. Please fix the issue.